ExampleMod/Common/Players/ExampleBiomePlayer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ExampleMod.Content.Biomes;
+using Microsoft.Xna.Framework.Graphics;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.Players
+{
+	public class ExampleBiomePlayer : ModPlayer
+	{
+		public override Texture2D GetMapBackgroundImage() {
```

Comment:
This should be fixed at the ModSceneEffect level to automatically happen.
This should not be a part of ModPlayer.